### ISSUE TITLE
RtmpCamera2 cannot restore original zoom

### ISSUE THREAD

**Issue by CubitSpeed** [pedroSG94/rtmp-rtsp-stream-client-java#670]
`RtmpCamera2` when setting `.setZoom(1.0)` will not restore the original zoom level. Instead, it is zoomed in.

`RtmpCamera1` does not have this issue and works as expected.

This can most easily be tested by changing `app/src/main/java/com/pedro/rtpstreamer/customexample/RtmpActivity.java` to use `RtmpCamera2`

(tested on multiple android devices)

**Comment by pedroSG94** [comment]
This should be fixed here:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/33fa37c0c958f715c5e8c7b6050cf59db9059652

**Comment by CubitSpeed** [comment]
Thank you! I appreciate the fast response!! It is fixed, but now I've realized there's a secondary issue (unsure if related). It looks like it was occurring before the change too. See #671

**Comment by CubitSpeed** [comment]
Just a quick note, setting zoom to `0.0f` will break things, instead I have to lower-end cap things at `0.1f` and then it's all good

**Comment by pedroSG94** [comment]
What do you mean? Do you want set min value to 0.1?
Do you get an error after zoom out?

**Comment by CubitSpeed** [comment]
> What do you mean? Do you want set min value to 0.1?
> Do you get an error after zoom out?

Sorry, let me be specific.

If you call `camera.setZoom(0f)`, the preview pane will freeze indefinitely (no error in console).

I mention setting to `0f` as this  "lower bound" you've set on this [line](https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/33fa37c0c958f715c5e8c7b6050cf59db9059652#diff-7c65bf82acb06ce1c68782f515191ab3R537). If instead of using `0f` as the lower bound, `0.1f` is used, there is no error.

In other words, I believe the correct line may be:
`if ((level <= maxZoom) && (level > 0)) {`
instead of:
`if ((level <= maxZoom) && (level >= 0)) {`

**Comment by pedroSG94** [comment]
Fixed: https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/8166559073582d8ed2a473b75c44a87eb04667fa
If you set zoom value out range, the result is min or max value zoom.

**Comment by CubitSpeed** [comment]
Awesome, thanks for the fix!
